packages/create-llama/templates/types/streaming/nextjs/app/api/chat/route.ts

```diff
@@ -42,7 +42,7 @@ export async function POST(request: NextRequest) {
     }
 
     const llm = new OpenAI({
-      model: process.env.MODEL || "gpt-3.5-turbo",
+      model: (process.env.MODEL as any) ?? "gpt-3.5-turbo",
```

Comment:
Is this necessary?

Reply:
Reduce `any` type as possible

Reply:
I had the same question. Problem is that model is of type 'keyof ALL_MODELS'. Any idea besides using any or exporting a type alias (means change to core?)

Reply:
oh, I've changed the type to `keyof ALL_MODELS | string` in the last commit, so it's not be ok